library/src/main/java/com/github/pwittchen/prefser/library/PreferencesAccessorsProvider.java

```diff
@@ -48,7 +48,7 @@ private void createAccessors() {
   private void createBooleanAccessor() {
     accessors.put(Boolean.class, new Accessor<Boolean>() {
       @Override public Boolean get(String key, Boolean defaultValue) {
-        return preferences.getBoolean(key, defaultValue);
+        return preferences.getBoolean(key, (defaultValue == null) ? false : defaultValue);
```

Comment:
The user will not get `true` or `false` if he/she passing `null`, the user will get `null`. Look the other comment I mention you, we are returning defValue before that check.
``` Java
if (!contains(key)) {
  return defaultValue;
}
```
Line 250 of Prefser.